jsdoc.conf.json

```diff
@@ -1,32 +1,33 @@
 {
-  "tags": {
-    "allowUnknownTags": true
+  "markdown": {
+    "hardwrap": true,
+    "parser": "gfm"
   },
-  "source": {
-    "include": ["lib/", "./docs/API.md", "bin"]
+  "mocha-docdash": {
+    "sort": true,
+    "static": false
   },
-  "plugins": ["plugins/markdown"],
   "opts": {
-    "encoding": "utf8",
-    "template": "node_modules/@mocha/docdash",
     "destination": "docs/api",
+    "encoding": "utf8",
     "recurse": true,
+    "template": "node_modules/@mocha/docdash",
+    "tutorials": "./docs/api-tutorials",
```

Comment:
outside this single line, I can't say I understand why any other changes were made here...

Reply:
I think I sorted the object keys.